### PR TITLE
Appveyor configuration

### PR DESCRIPTION
**Pull request by rasmushoj**
Appveyor configuration that tests C/ODBC against MariaDB 10.3.11.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+environment: 
+    matrix:
+        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+          DB: 10.3.11
```
Comment (lawrinn) [inline appveyor.yml]: Does it have to be concrete version? Will we always need to update the version here?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+platform: x64
+configuration: Release
+
+install:
+    
+
+build_script:
+    # build libmariadb separately first because otherwise the Wix installer build might look for files that aren't available yet
+    - cd libmariadb
+    - cmake --build . --config RelWithDebInfo --parallel 2
```
Comment (lawrinn) [inline appveyor.yml]: There is no need to build libmariadb - c/odbc's cmake does that

Comment (rasmushoj) [reply appveyor.yml]: I know, but there is some build parallelism problem in that case (at least in appveyor). That's why I build libmariadb separately first.

File: appveyor.yml (hunk 1 above)
Comment (rasmushoj) [reply appveyor.yml]: Yes, that is a problem, but I didn't yet figure out how to do it differently

**Pull request merged**